source/category/category.rst

```diff
@@ -1,13 +1,29 @@
 Graylog Message Categories 
 ============
 
+Categorization of messages is a fundamental building block of Illuminate. Categorization is the process of identifying and labelling messages by class, category, subcategory and type. The fields are defined as:
+
+* ``gim_event_type_code`` - This is the value assigned to an event log message that will be read by Illuminate core and used to enrich the remaining text fields with category information.
+* ``gim_event_class`` - Optional: Different categories can be group as a common class of events, where they may not share common fields but have some other association such as being related to endpoint activity.
+* ``gim_event_category`` - A GIM category is used to identify messages related to a common subject, such as authentication or http web traffic, which share a common set of fields.
+* ``gim_event_subcategory`` - The GIM subcategory is a more specific grouping of events based on common types of events, which use the fields defined in the parent category but can also use additional fields common to that subcategory.
+* ``gim_event_type`` - The specific type of event being categorized.
+
+The process of categorization involves the assignment of an event type code (See the table below for the mapping of type codes to categorical data) to the field ``gim_event_type_code``. This must be done prior to the execution of Illuminate core. When Core detects a message with a type code assignment in the field ``gim_event_type_code`` it will attempt to map the fields ``gim_event_class``, ``gim_event_category``, ``gim_event_subcategory``, and ``gim_event_type`` according to the assigned type code(s). 
+
+Illuminate core then 
```

Comment:
Was this supposed to be a complete statement or can it be removed?

Reply:
Removed, done.